src/nlc/NLC.json

```diff
@@ -33,6 +33,16 @@
 			"Please start training.",
 			"train"
 		  ]
-	    }
+		},
+		{
+			"class": "nlc.status",
+			"description": "Show the status for the natural language classifier",
+	  	"emittarget": "nlc.status.js",
+	  		"texts": [
+					"Can you provide the status for my classifier",
+					"Please obtain the status of classifier",
```

Comment:
Remove `Can you` and `Please` from the training statements because these aren't unique to any classification.